examples/simpletask/main.go

```diff
@@ -0,0 +1,70 @@
+package main
```

Comment:
I dont see the value of this example. Its not really doing anything. If you want to really test this move it into examples_test.go or call it func ExampleCompletion(...) in completion_test.go

Reply:
Hi, I do see its value, it's a simple yet quite realistic CLI app. It's quite useful for testing shell completions

Reply:
I find complete examples very useful. Sometimes you just need a bit of working code to debug something that doesn't work anymore. If completions break when migrating from v2 to v3, then using this v3 code as a reference, I could find the cause much faster.

Complete working examples are also useful for training AI.

It needs some better organization, though. Maybe even numbers to sort contents in the order people usually learn the library. By most frequent use cases.

https://github.com/urfave/cli/tree/ef45965eeb9c1122885fafa4a391b6be6a674f3d/examples

Reply:
Thats where the https://github.com/urfave/cli/blob/ef45965eeb9c1122885fafa4a391b6be6a674f3d/examples_test.go comes into play.

Reply:
@dearchap https://github.com/urfave/cli/blob/ef45965eeb9c1122885fafa4a391b6be6a674f3d/examples_test.go is not discoverable, and also they do not work if copy-pasted.

Reply:
`example/simpletask.go` doesn't tell anything to the person who browses examples. For the sake of bikeshedding `examples/commands.go` may be a better name.

I understand that this example can be used to test command completions, but because it is not used actually used in tests, I would suggest to submit it in a separate PR, where we could also discuss how to integrate it with what we have in docs.

Reply:
Good idea, I agree. I will add the `simpletask` example in a separate PR.

Reply:
Oh, actually why I like `/bin/bash`, is that it will refer to whatever is first in 

For example on macOS, the `/bin/bash` is a 20-years old bash3, and almost everyone using bash installs a modern bash5 with Homebrew.

Output on my machine:

```
$ which -a bash
/opt/homebrew/bin/bash
/bin/bash
```

So I think hardcoding `/bin/bash` in this case will always default to the pre-installed bash3, which I think is not so nice.

wdyt?